examples/framework-lit/src/components/Greeting.ts

```diff
@@ -0,0 +1,14 @@
+import {html, css, LitElement} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+
+@customElement('simple-greeting')
```

Comment:
Our custom Lit Vite plugin will also pick up on the `@customElement()` Lit decorator!